docs/learning/ECSQLTutorial/index.md

```diff
@@ -61,7 +61,7 @@ Or simply use the provided sample below:
 > SELECT * FROM bis.Element
 > ```
 
-<iframe class="embedded-console" src="/console/?imodel=House Sample&query=SELECT * FROM bis.Element"></iframe>
+<iframe class="embedded-console" src="/console/?imodel=House Sample Bak&query=SELECT * FROM bis.Element"></iframe>
```

Comment:
What's the Bak stand for?  Will the current docs be broken until this is released? If so, we should backport to 2.12.x branch and deploy from there.

Reply:
Recently had some projects rearranged in the samples project so renamed old House Sample to "House Sample Bak" for "backup". Current docs is broken until this is released. Will need to merge this into 2.12.x as well to get the fix asap.

Reply:
Sounds good, thanks.  If you add the comment `@Mergifyio backport release/2.12.x` it will automatically cherry-pick for you once the PR to master completes.